src/pages/components/card-section.mdx

```diff
@@ -47,6 +52,26 @@ In this variation, the last card in the Card group is set to high contrast and c
 
 Note: This uses the underlying component "Card group". You can find more information about that component <a href="https://www.ibm.com/standards/carbon/components/card-group">here</a>.
 
-To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
+For more information, see the [character count standards](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards).
+
+## Card section with images
+Card section with images brings images into each card in the section. Like Card section, this components is typically used for presenting resources or links, but also presents images in each card. When using Card section with media, avoid using a CTA card as this will cause the card heading to misalign.
```

Comment:
```suggestion
Card section with images brings images into each card in the section. Like Card section, this component is  used for presenting resources or links, but presents images instead of body copy in each card. 

When using this component, avoid using a CTA card as this will cause the card heading to misalign.
```